plugins/woocommerce-admin/client/inbox-panel/index.js

```diff
@@ -193,8 +233,15 @@ const InboxPanel = ( { showHeader = true } ) => {
 				'en_ZA',
 			];
 
+			const inboxQuery = {
+				...DEFAULT_INBOX_QUERY,
+				per_page: noteDisplayQty,
+			};
+			const notes = getNotes( inboxQuery );
+
 			return {
-				notes: getNotes( INBOX_QUERY ).map( ( note ) => {
+				notesHash: generateNotesHash( notes ),
+				notes: notes.map( ( note ) => {
```

Comment:
As mentioned below, a `map` inside of the `useSelect` is semi bad practice as it will break the memoization of the `useSelect` function. But again this is not something you did and was done prior.
Given we are only listening to the `notes` store within this `useSelect` it might not be as bad, but the preference would certainly be to move the map outside of the `useSelect`.

Reply:
Thanks! Done in https://github.com/woocommerce/woocommerce/pull/35003/commits/76a699601e389eacceb8f9d98b42e26d35683541.